tests/Aspire.Dashboard.Components.Tests/Controls/TextVisualizerDialogTests.cs

```diff
@@ -0,0 +1,152 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using Aspire.Dashboard.Components.Dialogs;
+using Aspire.Dashboard.Model;
+using Bunit;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.FluentUI.AspNetCore.Components;
+using Xunit;
+
+namespace Aspire.Dashboard.Components.Tests.Controls;
+
+public class TextVisualizerDialogTests : TestContext
+{
+    [Fact]
+    public async Task Render_TextVisualizerDialog_WithValidJson_FormatsJsonAsync()
+    {
+        var rawJson = """
+                      // line comment
```

Comment:
Any reason why we are changing the comment type as part of the formatting?

Reply:
JSON API doesn’t track different comment types. There isn’t anything we can easily do about it.